protocol/daemons/pricefeed/client/types/mutable_exchange_market_config_test.go

```diff
@@ -85,7 +85,7 @@ func TestGetMarketIds_Success(t *testing.T) {
 	}
 }
 
-func TestEqual_Mixed(t *testing.T) {
+func TestMutableExchangeMarketConfig_Equal(t *testing.T) {
```

Comment:
These had to be renamed due to identical testnames in this directory after file moves.